docs/en/Helm-based_deployment_guide.md

```diff
@@ -68,8 +63,6 @@ And create one more deployment, this time using a different set of overrides:
 
```

Comment:
../images/octopod_in_octopod_deployment.png wasn't updated for the latest web UI